jest.config.js

```diff
@@ -6,4 +6,7 @@ module.exports = {
   globals: {
     __VERSION__: require('./package.json').version,
   },
+  moduleNameMapper: {
+    axios: 'axios/dist/node/axios.cjs',
```

Comment:
Just curious: What makes this change necessary?

Reply:
axios is now published as ESM & CJS - after updating it - jest failed. This fixes it again.